src-backend/api/generator.py

```diff
@@ -8,6 +8,12 @@ def __init__(self, procedure):
         self.name = 'Procedure'
         self.procedure = procedure
 
+        if not self.procedure.title:
```

Comment:
is this testing if self.procedure.title is none? If so, why not use `if self.procedure.title is None:`